modules/monitoring-roles-for-monitoring-your-own-services.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="roles-for-monitoring-your-own-services_{context}"]
+= Roles for monitoring your own services
+
+This table lists the cluster roles deployed using the Cluster Monitoring Operator (CMO). Each of these roles can be bound against a specific namespace by the cluster administrator.
```

Comment:
"should" instead of "can"? cc @s-urbaniak

Reply:
yes correct, and even stronger. They _must_ be bound against the user namespace to have effect.